app/src/main/java/org/xbmc/kore/jsonrpc/HostConnection.java

```diff
@@ -101,6 +105,10 @@
         void onVolumeChanged(Application.OnVolumeChanged notification);
     }
 
+    public interface PlaylistNotificationsObserver {
+        void onPlaylistCleared(Playlist.OnClear notification);
+    }
+
```

Comment:
Pointed out previously: The other notifications should be implemented here, so that `HostConnectionObserver` doesn't need to poll Kodi if connection is through TCP.